tests/testthat/test-read_functions.R

```diff
@@ -1,24 +1,24 @@
 test_that("wtf_read_LI7810 works", {
 
+  options(whattheflux.quiet = TRUE)
```

Comment:
in test files, I myself tend to use `withr::local_options()` to be sure it's reset. :innocent:

Reply:
Thank you @maelle ! I'm hesitant to introduce another package dependency just for this, but will consider.